utils/spacewalk-common-channels.ini

```diff
@@ -910,15 +910,15 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Stable/images/repo/Uyuni-Server-4.0-POOL-x86_64-Media1/
 
-[uyuni-server-devel-leap-151]
+[uyuni-server-devel]
```

Comment:
Not sure about this one. Is the section name used for something ?

Remember what happens when we switch to leap 15.2 as base OS.
We may need to create a new section which share all what we have now and only change the `base_channel` to 15.2 . 
But we need unique names in in Uyuni for channels and repositories.

Reply:
> Not sure about this one. Is the section name used for something ?

AFAIK no. It's not at the doc (this is about master), and in the end we give the instructions to users about the URL: https://www.uyuni-project.org/pages/stable-version.html#serversetup

> Remember what happens when we switch to leap 15.2 as base OS.
> We may need to create a new section which share all what we have now and only change the base_channel to 15.2 .
> But we need unique names in in Uyuni for channels and repositories.

Well, in the end when we switch to 15.2 as base OS (not done yet!), then we will not have 15.1, and 15.2 will be required (the product will change to show this, the sections here will be updated, and users should recreate their environments (in the end nobody should be using devel/master for production).

Reply:
If nobody use it, why do we have it there?

But as you said. Master is not the big problem. We need to think about Stable and how we handle it. The users need to be able to update and switch the channels.

Reply:
> If nobody use it, why do we have it there?
If talking about the server: for consistency, as we have the spacewalk server as well (and our proxy). But I think we could very well remove it.

If talking about the proxy: yes, that one is used, as users want the proxy channel available for the proxies.

> But as you said. Master is not the big problem. We need to think about Stable and how we handle it. The users need to be able to update and switch the channels.

My 2 cents:

1. Static URL repositories, as we have now for Master. In the end we use `osc release` so the URL will be the same we have for master, but changing `Master` with `Stable`. Benefit: nobody needs to change the `.repo` file.
2. The package `Uyuni-Server-release` has the following dependencies right now:
    ```
    uyuni-master-srv:~ # rpm -q --requires Uyuni-Server-release
    product(openSUSE) >= 15.1
    rpmlib(CompressedFileNames) <= 3.0.4-1
    rpmlib(FileDigests) <= 4.6.0-1
    rpmlib(PayloadFilesHavePrefix) <= 4.0-1
    rpmlib(PayloadIsXz) <= 5.2-1
    ```
    so when we change the product to depend on openSUSE Leap 15.2, users will not be able to update Uyuni if they don't upgrade openSUSE first (so they will not run into troubles).